internal/fullnode/labels_test.go

```diff
@@ -0,0 +1,30 @@
+package fullnode
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+
+	cosmosv1 "github.com/strangelove-ventures/cosmos-operator/api/v1"
+)
+
+func TestDefaultLabels(t *testing.T) {
+	t.Parallel()
+
+	t.Run("give full node", func(t *testing.T) {
```

Comment:
One more test case for blank `""` which defaults to FullNode.

Reply:
But best top put that test case somewhere else, like pod builder because it tests the public interface.